ros_cross_compile/platform.py

```diff
@@ -26,14 +26,10 @@
 }
 SUPPORTED_ARCHITECTURES = tuple(ARCHITECTURE_NAME_MAP.keys())
 
-SUPPORTED_ROS2_DISTROS = ('dashing', 'eloquent', 'foxy', 'galactic', 'rolling')
-SUPPORTED_ROS_DISTROS = ('kinetic', 'melodic')
+SUPPORTED_ROS2_DISTROS = ('dashing', 'foxy', 'galactic', 'rolling')
```

Comment:
It's fine to remove these from the e2e tests since they're EOL, but I don't see any reason to remove them from the allowable inputs - a user could still build Eloquent or Kinetic from source, if they wanted to (though I wouldn't generally advise it)